### PR TITLE
added tests for clickhouse database

### PR DESCRIPTION
**Pull request by prinkov**
Hi!
I want added tests for clickhouse database

### CODE REVIEW

File: database/clickhouse/clickhouse.go [hunk 1]
```diff
@@ -58,8 +58,9 @@ func WithInstance(conn *sql.DB, config *Config) (database.Driver, error) {
 }
 
 type ClickHouse struct {
-	conn   *sql.DB
-	config *Config
+	conn     *sql.DB
+	config   *Config
+	isLocked bool
```
Comment (dhui) [inline database/clickhouse/clickhouse.go:63]: Use https://pkg.go.dev/go.uber.org/atomic@v1.6.0#NewBool

File: database/clickhouse/clickhouse_test.go [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+			}
+		}()
+
+		dt.Test(t, d, []byte("SELECT 1"))
+
+		if err := d.(*ClickHouse).ensureVersionTable(); err != nil {
+			t.Fatal(err)
+		}
+
+		if err := d.(*ClickHouse).ensureVersionTable(); err != nil {
```
Comment (dhui) [inline database/clickhouse/clickhouse_test.go:90]: Why is `ensureVersionTable()` called twice? Does it need to be called at all?

File: database/clickhouse/clickhouse_test.go [hunk 3]
```diff
@@ -0,0 +1,184 @@
+package clickhouse
```
Comment (dhui) [inline database/clickhouse/clickhouse_test.go:1]: Can this be `clickhouse_test`?

**Review by dhui — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: database/clickhouse/clickhouse.go (hunk 1 above)
Comment (prinkov) [reply database/clickhouse/clickhouse.go]: thnx, used

File: database/clickhouse/clickhouse_test.go (hunk 2 above)
Comment (prinkov) [reply database/clickhouse/clickhouse_test.go]: yes, you right. I removed it, thanks

File: database/clickhouse/clickhouse_test.go (hunk 3 above)
Comment (prinkov) [reply database/clickhouse/clickhouse_test.go:1]: this test included in the package clickhouse, not clichouse_test. Other databases tests included in databases packages too

Comment (dhui) [reply database/clickhouse/clickhouse_test.go:1]: I didn't see any usage of unexported identifiers, but could have missed some. If we don't need to use unexported identifiers in tests, let's use a different package name for the tests and import the `clickhouse` package.

> Other databases tests included in databases packages too

Some db tests test unexported methods. We should really split our tests into internal and external tests...

Comment (prinkov) [reply database/clickhouse/clickhouse_test.go]: I understood. Thanks, fixed

**Review by dhui — APPROVED** [review]
Thanks for address the PR feedback!

**Pull request merged**